SmartSyncExplorerSwift/Podfile

```diff
@@ -0,0 +1,23 @@
+platform :ios, '10.0'
+
+project 'SmartSyncExplorerSwift.xcodeproj'
+target 'SmartSyncExplorerSwift' do
+
+source 'https://github.com/CocoaPods/Specs.git'
+
+use_frameworks!
+
+pod 'SalesforceAnalytics', :path => 'mobile_sdk/SalesforceMobileSDK-iOS'
+pod 'SalesforceSDKCore', :path => 'mobile_sdk/SalesforceMobileSDK-iOS'
+pod 'SmartStore', :path => 'mobile_sdk/SalesforceMobileSDK-iOS'
+pod 'SmartSync', :path => 'mobile_sdk/SalesforceMobileSDK-iOS'
+pod 'SalesforceSwiftSDK', :path => 'mobile_sdk/SalesforceMobileSDK-iOS'
+pod 'PromiseKit', :git => 'https://github.com/mxcl/PromiseKit', :tag => '5.0.3'
```

Comment:
Doesn't SalesforceSwiftSDK depend on PromiseKit? So why do we need to list it here.

Reply:
@wmathurin  This was generated from the [native swift template](https://github.com/forcedotcom/SalesforceMobileSDK-Templates/blob/master/iOSNativeSwiftTemplate/Podfile). I think the podspec for the SalesforceSwiftSDK will have to be corrected and tested. Will do in a different PR.